Gemfile

```diff
@@ -77,7 +77,7 @@ gem 'ruby-progressbar'
 
 # own gems
 gem 'quintel_merit', ref: '421f3fb', github: 'quintel/merit'
-gem 'atlas',         ref: 'f7aebb9', github: 'quintel/atlas'
+gem 'atlas',         ref: '5e85101', github: 'quintel/atlas'
```

Comment:
FYI: For updating Atlas we always use the same commit message: "Bump Atlas to quintel/atlas@5e85101"

Also if you work with Atlas locally, we normaly push Atlas and update the ref here instead of pushing your path 🙂